content/2018-munich/speakers/julius-volz.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Julius Volz
+---
+
+## Julius Volz
+
+Julius co-founded Prometheus and lead the project to success at SoundCloud and beyond. He now freelances around Prometheus, focuses on growing the Prometheus community, and is the lead organizer of PromCon. In a previous life, Julius was a Site Reliability Engineer at Google.
```

Comment:
"is the lead organizer of PromCon" is no longer true this year... "and founded PromCon"?

Reply:
I'm not sure PromCon has one founder.

Reply:
Well I never want to claim that PromCon would have been possible without the help of the Prometheus team (especially RP and its money-handling, for which it was compensated). But given that I'm the guy who had the idea, got the domain, then planned and executed around 90% of the conference by myself (especially in the first months), I think calling myself the founder is legitimate.

Reply:
I just copied it over from 2017; will fix.

Re specific wording: as the two of you are orthogonal, I am not sure with what to fix it.

Reply:
My recollection honestly isn't that clear on the matter.

Reply:
While I feel strongly about being *the* founder of PromCon and will continue saying that, I am happy in this instance to change "founded" to "started" if that helps unblock this PR.

Reply:
I don't mind too much either way.

Reply:
Ok, well then put "founded" :)